src/content/docs/browser/new-relic-browser/troubleshooting/troubleshoot-ajax-data-collection.mdx

```diff
@@ -45,54 +46,79 @@ If your application is [instrumented with <InlinePopover type="browser"/>](/docs
     // fetch 
     ƒ fetch() { [native code] }
     ```
-
+    <Callout variant="tip">
     If you see this type of failure or you see a different response, AJAX instrumentation may be disabled or you may be using another script or library that is conflicting with New Relic instrumentation.
-  </Collapser>
-
-  <Collapser
-    id="verify-ajax-enabled"
-    title="2. Verify AJAX instrumentation is enabled."
-  >
-    AJAX instrumentation is not included in the lite agent. Open your browser console dev tool and check the `newrelic.initializedAgents` object. This object contains an entry for every agent running on the page. There is typically only one agent on a page. Inspect the initialized agent object for a `runtime` object. This object contains information about the running agent like `loaderType`. If `loaderType` is `lite`, AJAX instrumentation is not included in the agent being used.
-
-    If the `loaderType` is not `lite`, you can also check the initialized agent for `config` and `features` objects. The `config` object will contain runtime configuration of the agent, including an `ajax` object with an `enabled` boolean. The `features` object should contain an entry for each feature initialized by the agent, including an `ajax` entry. Verify this ajax object contains a boolean `enabled` of `true` and `featAggregate` entry. If any of these items are not true, the AJAX instrumentation may not have initialized due to a configuration change in New Relic on the browser entity or possibly a manual change to the `NREUM.init` object that was copy/pasted into the HTML.
-  </Collapser>
+    </Callout>
+
+  </Step>
+
+
+  <Step>
+
+    ## Verify AJAX instrumentation is enabled
+
```

Comment:
@adutta-newrelic I like how you broke up those long paragraphs! Still, I think you could make this a bit easier to follow by converting these into steps. What do you think about an approach like this?

 ## Verify AJAX instrumentation is enabled

 AJAX instrumentation is not included in the lite agent. 
 
 To verify Ajax instrumentation:
    
   1. Open your browser console dev tool and check the `newrelic.initializedAgents` object. This object contains an entry for every agent running on the page. There is typically only one agent on a page. 
   
   2. Inspect the initialized agent object for a `runtime` object. This object contains information about the running agent like `loaderType`. If `loaderType` is `lite`, AJAX instrumentation is not included in the agent being used.

   3. If the `loaderType` is not `lite`, you can also check the initialized agent for `config` and `features` objects. 
    
       * The `config` object will contain runtime configuration of the agent, including an `ajax` object with an `enabled` boolean.     
       * The `features` object should contain an entry for each feature initialized by the agent, including an `ajax` entry. 
    
   4. Verify this AJAX object contains a boolean `enabled` of `true` and `featAggregate` entry. 
    
    If any of these items are not true, the AJAX instrumentation may not have initialized due to a configuration change in New Relic on the browser entity or possibly a manual change to the `NREUM.init` object that was copy/pasted into the HTML.